modules/mockk/src/commonMain/kotlin/io/mockk/MockK.kt

```diff
@@ -1,8 +1,70 @@
+import kotlin.reflect.KClass
+
+
+
+private val nonMockableClasses = listOf(
+
+  System::class,
+
```

Comment:
I think you have a code formatter that adds a ton of newlines. Can you please not add needless empty lines?

Reply:
Sure i will remove em